modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js

```diff
@@ -170,6 +170,7 @@ const Autocomplete = ({
 }) => {
 	const [value, setValue] = useSyncValue(initialValue, syncDelay);
 	const [visible, setVisible] = useState(false);
+	const [selectedOption, setSelectedOption] = useState(false);
```

Comment:
@mlan0, good job! Can you please use the word `item` instead of `option` once this is the term already in use in the existing code?

Thanks.